### DISCUSSION TITLE
Dokumentacja API https://elicznik.tauron-dystrybucja.pl/energia/api

### DISCUSSION THREAD

**Discussion by darkowalski** [PiotrMachowski/Home-Assistant-custom-components-Tauron-AMIplus#125] [Q&A]
Panie @PiotrMachowski , czy posiada Pan dokumentację do API https://elicznik.tauron-dystrybucja.pl/energia/api ?

Ewentualnie gdzie można uzyskać wspomnianą dokumentację?

Pozdrawiam

**Comment by PiotrMachowski** [comment]
To nie jest "prawdziwe" API, tylko wewnętrzne, używane przez ich własną stronę, także nie ma (i nie będzie) do niego żadnej dokumentacji.

**Reply by darkowalski** [reply]
Podpowie Pan w którym elemencie strony Tauron to API zostaje użyte? Nigdzie nie mogę się tego doszukać, a wszelkie formularze z żądaniami POST odwołują się do standardowych kontrolerów, nie do wspomnianego API. 

Z góry dziękuję
